setup.cfg

```diff
@@ -71,8 +72,10 @@ all =
 testing =
     coverage
     hypothesis
+    napari[pyqt5]!=0.4.18
```

Comment:
Can you add a comment here about why we blocked 0.4.18? If you remember, I've totally forgotten! 😅